test/config/jobconfig.json

```diff
@@ -1,53 +1,80 @@
-[
+{
+  "configVersion": "v1.0 2024-03-01 6f3f38",
+  "jobs": [
   {
-    "jobType": "archive",
+    "jobType": "all_access",
+    "configVersion": "v1.0 2024-03-01 6f3f38",
```

Comment:
There is a `configVersion` per job but also a general one at the top of the file. We should only keep the top one. the current implementation expects only that and then stores this information inside each job, but it's not needed to duplicate it per job in the configuration file.

Reply:
Resolved in the latest commit